unicode-transforms.cabal

```diff
@@ -42,12 +42,8 @@ extra-source-files:
     stack.yaml
     stack-7.10.yaml
     stack-8.0.yaml
-    unicode-data/README.md
-    unicode-data/ucd/NormalizationTest.txt
-    unicode-data/extra/NormalizationTest.txt
-    unicode-data/stack.yaml
-    unicode-data/ucd2haskell.cabal
-    unicode-data/UCD2Haskell.hs
+    test/data/NormalizationTest.txt
+    test/data/extra/NormalizationTest.txt
```

Comment:
Do we have instructions somewhere about how to update these files for a new unicode version?

Reply:
No, we are not. We should probably mention that in the docs. I'm merging this now. The docs can be modified accordingly, once we release `unicode-data`.